storagenode/storagenodedb/bandwidthdb.go

```diff
@@ -220,6 +267,110 @@ func (db *bandwidthDB) Rollup(ctx context.Context) (err error) {
 	return nil
 }
 
+// GetDailyRollups returns slice of daily bandwidth usage rollups for provided time range,
+// sorted in ascending order.
+func (db *bandwidthDB) GetDailyRollups(ctx context.Context, from, to time.Time) (_ []bandwidth.UsageRollup, err error) {
+	defer mon.Task()(&ctx, from , to)(&err)
+
+	since, _ := date.DayBoundary(from.UTC())
+	_, before := date.DayBoundary(to.UTC())
+
+	return db.getDailyUsageRollups(ctx,
+		"WHERE DATETIME(?) <= DATETIME(interval_start) AND DATETIME(interval_start) <= DATETIME(?)",
+		since, before,
+		since, before)
```

Comment:
Here it's not clear how the the extra arguments are handled. The query only uses two arguments however there are four passed in. Maybe there's a way to avoid the duplication?

It seems that the args are always the same so, might as well duplicate them inside `getDailyUsageRollups` rather than outside.